src/dynamo-streams.ts

```diff
@@ -70,6 +80,22 @@ export type TestEvent<Entity> = {
   records: TestRecord<Entity>[];
 };
 
+/**
+ * Stringifies an unmarshalled DynamoDB key deterministically, regardless
+ * of ordering of keys.
+ */
+const deterministicStringify = (obj: {
```

Comment:
As this method doesn't really care what comes in, changing the type to `Record<string, any>` would eliminate the linting issue below.

Reply:
Sorry, the linting issue is actually for the unmarshall statement.  But they do go together.  line 208

Reply:
I'm going to inline this function _and_ remove unmarshalling, which should address the concern.